src/resolvers/Query/getDonationByOrgIdConnection.ts

```diff
@@ -1,13 +1,11 @@
-import {
-  DonationWhereInput,
-  InputMaybe,
-  QueryResolvers,
-} from "../../types/generatedGraphQLTypes";
-import { Donation } from "../../models";
+import { QueryResolvers } from "../../types/generatedGraphQLTypes";
+import { Donation, Interface_Donation } from "../../models";
+import { FilterQuery } from "mongoose";
+import { getInputArgs } from "./helperFunctions/getInputArgs";
 
 export const getDonationByOrgIdConnection: QueryResolvers["getDonationByOrgIdConnection"] =
   async (_parent, args) => {
-    const inputArg = getInputArg(args.where);
+    const inputArg: FilterQuery<Interface_Donation> = getInputArgs(args.where);
```

Comment:
This is how the generic is supposed to be used:-
```ts
const inputArgs = getInputArgs<Interface_Donation>(args.where);
```
Change this in all the resolver implementations.

Reply:
Done